### PR TITLE
Destination BigQuery: CDK updates for latest changes

### PR DESCRIPTION
**Pull request by gisripa**
## What
* Upgrade to latest CDK to fix flaky tests

## How
* See [CDK Updates](https://github.com/airbytehq/airbyte/pull/34728)

### CODE REVIEW

**Comment by gisripa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34728?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* `master`
  * **#34727** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34727?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **#34728** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34728?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gisripa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java [hunk 1]
```diff
@@ -52,6 +53,11 @@ public Optional<TableDefinition> findExistingTable(final StreamId id) {
     return Optional.ofNullable(table).map(Table::getDefinition);
   }
 
+  @Override
+  public LinkedHashMap<String, TableDefinition> findExistingFinalTables(List<StreamId> streamIds) throws Exception {
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java:57]: shouldn't this at least delegate to `findExistingTable`? or is this method just unused right now

**Review by edgao — APPROVED** [review]
one question, lgtm assuming I'm misunderstanding current state of the world

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java (hunk 1 above)
Comment (gisripa) [reply airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java:57]: unusued, WIP.

**Comment by gisripa** [comment]
### Merge activity

* **Feb 1, 3:14 PM EST**: @gisripa merged this pull request with [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/34728).

**Pull request merged**